packages/blocks/src/lib/navigation-map/components/vector-input.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang='ts'>
+
+import { Label, SliderInput } from '@viamrobotics/prime-core';
+import { createEventDispatcher } from 'svelte';
+
+export let readonly = false;
+export let placeholders = ['0', '0', '0'];
+export let labels = ['x', 'y', 'z'];
+export let type: 'integer' | 'number' = 'number';
+export let values: number[] = [];
+export let step = 1;
+
+const inputs: HTMLInputElement[] = [];
```

Comment:
Should this be an object keyed by `label` rather than an array? If the `labels` prop changes, this feels like it will break

Reply:
Good call - fixed!